security/cert_manager_operator/index.adoc

```diff
@@ -0,0 +1,26 @@
+[id="cert-manager-operator-about"]
+= About the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-operator-about
+
+toc::[]
+
+TODO
+
+// Make sure to add heading to appear in TOC
+
+////
+Things to cover
+ - General about
+ - Use cases
+ - Diff between this and the cert manager already in OCP
```

Comment:
@slaskawi I'll check the use cases doc and other places to confirm, but is the main difference between the two is that the cert-manager that ships with OCP is used for certs for the other core OCP components, and then this one is for administering certs for users' own workloads/custom needs?

Reply:
No, currently OCP does not ship cert-manager by default. It ships so-called Service CA Operator, which is also capable of issuing certificates for core components and user workloads. Both operators (cert-manager and Service CA) are two different things.

Reply:
@slaskawi so you're saying that we shouldn't mention that service CA operator at all then here?